google-cloud-pubsub/OVERVIEW.md

```diff
@@ -464,54 +480,6 @@ sub.acknowledge received_messages
 sub.seek snapshot
 ```
 
-## Listening for Messages
```

Comment:
I consolidated these into the `Receiving Messages` header above, since this block seemed repetitive and more useful above.

Reply:
Thank you for finding and removing this duplicated content!